src/CTA.WebForms2Blazor/FileConverters/ViewFileConverter.cs

```diff
@@ -66,23 +73,25 @@ private void FindConversionActions(HtmlNode node, HtmlNode parent)
 
         private void GetActions(HtmlNode node, HtmlNode parent)
         {
+            string controlConverterType = "";
             if (SupportedControls.ControlRulesMap.ContainsKey(node.Name))
             {
                 var conversionAction = new ControlConversionAction(node, parent, SupportedControls.ControlRulesMap[node.Name]);
+                controlConverterType = conversionAction.ControlConverter.GetType().Name;
                 _controlActions.Add(conversionAction);
-            }
+            } 
             else if (SupportedControls.UserControls.UserControlRulesMap.ContainsKey(node.Name))
             {
-                var conversionAction = new ControlConversionAction(node, parent,
-                    SupportedControls.UserControls.UserControlRulesMap[node.Name]);
+                var conversionAction = new ControlConversionAction(node, parent, SupportedControls.UserControls.UserControlRulesMap[node.Name]);
+                controlConverterType = conversionAction.ControlConverter.GetType().Name;
                 _controlActions.Add(conversionAction);
             }
-            else
+            else if(UnknownControlRemover.ControlStartTagRegex.IsMatch(node.Name) || UnknownControlRemover.ControlStartTagRegex.IsMatch(node.Name))
```

Comment:
The regexes `ControlStartTagRegex` and `ControlEndTagRegex` matches the entire tag including the `<>` and `</>` brackets along with any attributes whereas `node.Name` does not and only includes the name of the tag. For this reason, this condition should never be true. Also, you are using `ControlStartTagRegex` on both sides of the `||`.